go/sign/access_key_signer_test.go

```diff
@@ -0,0 +1,40 @@
+package sign
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestAccessKeyHTTPSigner_Sign(t *testing.T) {
+	url := "https://gateway-staging.ncrcloud.com/site/sites/find-nearby/88.05,46.25?radius=10000"
+	req, _ := http.NewRequest("GET", url, strings.NewReader(""))
+	req.Header.Add("Date", time.Now().UTC().Format(http.TimeFormat))
+	req.Header.Add("nep-organization", "")
+	req.Header.Add("Content-Type", "application/json")
+
+	// Replace the empty string with your shared key
+	sharedKey := ""
+	// Replace the empty string with your secret key
+	secretKey := ""
+	s, _ := NewAccessKeyHTTPSigner(sharedKey, secretKey)
+	req, err := s.Sign(req)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Prefer t.Fatal / t.Fatalf to indicate test case failure and end test case execution.

Reply:
Good catch I'll update it